package.json

```
@@ -29,11 +29,15 @@
     "docs:build": "npx typedoc",
     "docs:build:watch": "npm run docs:build -- --watch",
     "docs:serve": "PORT=4000 npx serve --no-clipboard docs",
-    "test": "npx jest",
-    "test:ci": "npm run test -- --ci --verbose false",
-    "test:watch": "npm run test -- --watchAll",
-    "test:e2e": "npm run test -- --config __tests__/e2e/jest.e2e-config.js",
-    "test:e2e:watch": "npm run test -- --config __tests__/e2e/jest.e2e-config.js --watchAll",
+    "test:unit": "npx jest",
+    "test:ci": "npm run test:unit -- --ci --verbose false && npm run test:e2e -- --ci --verbose false",
```

Not sure about the `false`. If your intention to remove `verbose` just omit from the argument list.

```suggestion
    "test:ci": "npm run test:unit -- --ci --verbose && npm run test:e2e -- --ci --verbose",
```